mail_activity_done/hooks.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# Copyright 2018 Odoo, S.A.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo.addons.mail.models.mail_activity import MailActivity
+
+
+def pre_init_hook(cr):
+    """
+    The objective of this hook is to default to false all values of field
+    'done' of mail.activity
+    """
+    cr.execute("""SELECT column_name
+    FROM information_schema.columns
+    WHERE table_name='mail_activity' AND
+    column_name='done'""")
+    if not cr.fetchone():
+        cr.execute(
+            """
+            ALTER TABLE mail_activity ADD COLUMN done boolean;
```

Comment:
What happens if the table is empty but already has the column `done`?

Reply:
If the column exists it won't enter the if clause and won't create the column, right?

Reply:
right 👍